pajbot/modules/emote_limit.py

```diff
@@ -70,6 +70,15 @@ class EmoteLimitModule(BaseModule):
             default="Too many emotes in your message",
             constraints={},
         ),
+        ModuleSetting(
+            key="whisper_timeout_reason",
+            label="Whisper Timeout Reason | Available arguments: {timeout_duration}",
+            type="text",
+            required=False,
+            placeholder="",
+            default="You have been timed out for {timeout_duration} for posting too many emotes.",
```

Comment:
If anything, you missed a "seconds" here

```suggestion
            default="You have been timed out for {timeout_duration} seconds for posting too many emotes.",
```

Reply:
Huh? I thought I added that